src/core/opamStd.ml

```diff
@@ -1370,7 +1386,7 @@ module Config = struct
     in
     OpamCoreConfig.(setk (setk (fun c -> r := c; k)) !r)
       ?debug_level:(env_level "DEBUG")
-      ?debug_sections:None
+      ?debug_sections:(env_sections "DEBUGSECTIONS")
```

Comment:
One last thing, it need to be added in [doc](https://github.com/ocaml/opam/blob/master/src/client/opamArg.ml#L238).

Reply:
Done!